### DISCUSSION TITLE
Feature Request: Hide buttons dynamically in horizontal button stack

### DISCUSSION THREAD

**Discussion by ngrmr** [Clooos/Bubble-Card#490] [Feature requests]
First of all I have to thank you soo much for your work - I just started with Home Assistant and I am simply overwhelmed by the possibilites you have compared to the closed SW solutions. Your bubble cards are helping me sooo much to set up a nice and clean looking mobile dashboard.
Maybe I am using the horizontal stack different compared to the rest of you guys, but I really only need a small dashboard for my flat with some scene toggles. Thats why I wanted to implement some popups for more detailed controll if needed.
That generally works pretty good already, but it would be so neat if I could only show the #lights-livingroom popup once my presence is detected in the living room. If I am in the bedroom it would be replaced by the #lights-bedroom popup. 
Maybe this can even be achieved today by just pushing the button out of the visable area, but if there is a chance to hide the button based on conditions it would make a nice update to your already great SW.

![Screenshot 2024-04-28 at 13 02 06](https://github.com/Clooos/Bubble-Card/assets/166990364/a117afeb-81d3-4025-826f-9ab7f305869d)

**Comment by Clooos** [comment]
Hi, I'm not sure to understand but you can use the pop-up trigger to open a pop-up based on a presence sensor. This way you can always have the right pop-up opened when you are entering in a room, the only downside is that it only really works if you are alone in your flat.

**Reply by ngrmr** [reply]
Yes, I know that. But I think it would be nice to only have 3 buttons in the horizontal stack: the colour picker, a "universal" light card [#lights-livingroom when I am in the living room and #lights-bedroom when I am in the bedroom] and a "universal" shades card [#shades-livingroom and #shades-bedroom]. In order to achieve these "universal" cards I could create all of them and just hide #livingroom when I am not there. Do you get the idea? 
As you said I could in theory already achieve something similar, but it would look way cleaner if the buttons were hidden. And currently I would only get the option to have one room specific popup if I am not wrong.

**Reply by Clooos** [reply]
You can probably do that with multiples horizontal buttons stack in conditional cards, this way you will always have the right buttons.

**Reply by ngrmr** [reply]
Thanks for the hint - I will have a look into this

**Reply by iemand-dev** [reply]
@Clooos 
I'd like to argument my use case
I want to use the bubble cards to get access to room controles when there is presence detect, but also for tasks. I create a helper in HA and set this helper as "presence detected" when a task must be completed (like taking out the trash, setting the planning in the charging of my car or low battery warnings of sensors). However now I see a lot of buttons based on all task groups which I don't really want to see.
The order (based on date changed) helps, but still there are way too many buttons compared to what really is needed.

I'd really like the option to hide the button if the presence sensor does not detect anything.
